src/javafundamentals/interfaces/Ex01/CarsForSale.java

```diff
@@ -0,0 +1,13 @@
+package javafundamentals.interfaces.Ex01;
+
+public class CarsForSale implements SalesMethods{
+    @Override
+    public void onlinePhotos() {
```

Comment:
Method names should be actions(verbs), not properties(nouns).  
if this method was intended to print the photos, then a better name would be:
`printPhotos()`

Reply:
That makes sense! thanks!